portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/clay/ClayAlertToast.testcase

```diff
@@ -72,6 +72,31 @@ definition {
 		}
 	}
 
+	@description = "LPS-170575 - Verify toast alert with no actions disappears after 5 seconds"
+	@priority = "5"
+	test DisappearsAfter5SecsWithoutActions {
+		task ("When a toast alert appears") {
+			Click(
+				key_text = "DisappearsAfter5Secs",
+				locator1 = "Button#ANY");
+		}
+
+		task ("And no actions on alert") {
+			VerifyElementNotPresent(locator1 = "//*[@id='ToastAlertContainer']//a");
```

Comment:
I think I see why you set `_dismissible` now...I guess that works but really we want to verify that there's no other buttons (actions) on the toast. As we discussed, the close button can be present and it should still work. Could you verify if that's still the case locally?

Reply:
I don't know if I understand what you're asking, so do you think it would be better to change the value of locator1 in this case?

Reply:
locally the test works fine.